sdk/identity/identity/src/credentials/managedIdentityCredential/cloudShellMsi.ts

```diff
@@ -53,14 +53,16 @@ function prepareRequestOptions(
 
 /**
  * Defines how to determine whether the Azure Cloud Shell MSI is available, and also how to retrieve a token from the Azure Cloud Shell MSI.
+ * Since Azure Managed Identities aren't available in the Azure Cloud Shell, we log a warning for users that try to access cloud shell using user assigned identity.
  */
 export const cloudShellMsi: MSI = {
-  async isAvailable(scopes): Promise<boolean> {
+  async isAvailable(scopes, _identityClient): Promise<boolean> {
```

Comment:
```suggestion
  async isAvailable(scopes): Promise<boolean> {
```
This shouldn’t be needed anymore 🙂

Reply:
done